doc/classes/PacketPeerUDP.xml

```diff
@@ -4,7 +4,33 @@
 		UDP packet peer.
 	</brief_description>
 	<description>
-		UDP packet peer. Can be used to send raw UDP packets as well as [Variant]s.
+		UDP packet peer. Can be used to send and receive raw UDP packets as well as [Variant]s.
+		To send a packet:
+		[codeblock]
+		var peer = PacketPeerUDP.new()
+
+		# OPTIONAL: Select local port that will be used to send the packet.
+		peer.bind(4444)
+
+		peer.set_dest_address("1.1.1.1", 4433)
+		peer.put_packet("hello".to_utf8_buffer())
+		[/codeblock]
+		To listen for packets:
+		[codeblock]
+		var peer
+
+		func _ready():
+		    peer = PacketPeerUDP.new()
+		    peer.bind(4433)
+		
+		
```

Comment:
As already mentioned on [RocketChat](https://chat.godotengine.org/channel/documentation?msg=vhqPgtqR6qkHK5jAH), it's likely the static checks fail because these empty lines contain tabulation characters (GitHub doesn't always show this).